### PR TITLE
Add citation for JuMP 1.0 paper

### PR DESCRIPTION
**Pull request by odow**

### CODE REVIEW

File: CITATION.bib [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
-  pages={295--320},
-  year={2017},
-  publisher={SIAM}
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Benoît Legat and Juan Pablo Vielma},
+    title = {JuMP 1.0: Recent improvements to a modeling language for mathematical optimization},
+    journal = {Mathematical Programming C},
+    year = {2023},
+    note = {To appear.}
+    doi = {10.48550/arXiv.2206.03866}
```
Comment (odow) [inline CITATION.bib:7]: Note sure exactly what I should put here?

File: CITATION.bib [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
-  number={2},
-  pages={295--320},
-  year={2017},
-  publisher={SIAM}
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Benoît Legat and Juan Pablo Vielma},
+    title = {JuMP 1.0: Recent improvements to a modeling language for mathematical optimization},
+    journal = {Mathematical Programming C},
+    year = {2023},
+    note = {To appear.}
```
Comment (mlubin) [inline CITATION.bib:6]: "In press" maybe?

File: CITATION.bib [hunk 3]
```diff
@@ -1,10 +1,8 @@
...
-  journal={SIAM Review},
-  volume={59},
-  number={2},
-  pages={295--320},
-  year={2017},
-  publisher={SIAM}
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Benoît Legat and Juan Pablo Vielma},
+    title = {JuMP 1.0: Recent improvements to a modeling language for mathematical optimization},
+    journal = {Mathematical Programming C},
```
Comment (mlubin) [inline CITATION.bib:4]: "Mathematical Programming Computation" is the official name.

File: CITATION.bib (hunk 1 above)
Comment (mlubin) [inline CITATION.bib:7]: Better to wait until we have the real DOI?

File: README.md [hunk 4]
```diff
@@ -52,38 +52,48 @@ feature requests, etc.
...
-    pages = {238-248},
-    year = {2015},
-    doi = {10.1287/ijoc.2014.0623},
-}
-```
-
-A preprint of this paper is [freely available](https://arxiv.org/abs/1312.1431).
+For earlier works, see:
+
+ * Our paper in SIAM Review ([journal](https://dx.doi.org/10.1137/15M1020575), [preprint](https://mlubin.github.io/pdf/jump-sirev.pdf)):
```
Comment (mlubin) [inline README.md:70]: The linked pdf is a final version, not a preprint.

File: CITATION.bib (hunk 3 above)
Comment (odow) [inline CITATION.bib:4]:
```suggestion
    journal = {Mathematical Programming Computation},
```

File: CITATION.bib (hunk 2 above)
Comment (odow) [inline CITATION.bib:6]:
```suggestion
    note = {In press.}
```

File: README.md (hunk 4 above)
Comment (odow) [inline README.md:70]:
```suggestion
 * Our paper in SIAM Review ([journal](https://dx.doi.org/10.1137/15M1020575), [pdf](https://mlubin.github.io/pdf/jump-sirev.pdf)):
```

File: docs/src/index.md [hunk 5]
```diff
@@ -83,39 +83,19 @@ you know where to look for certain things.
...
-doi = {10.1137/15M1020575},
+following paper ([preprint](https://arxiv.org/abs/2206.03866)):
+
+```bibtex
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Benoît Legat and Juan Pablo Vielma},
+    title = {JuMP 1.0: Recent improvements to a modeling language for mathematical optimization},
+    journal = {Mathematical Programming C},
+    year = {2023},
+    note = {To appear.},
```
Comment (odow) [inline docs/src/index.md:94]:
```suggestion
    note = {In press.}
```

File: docs/src/index.md [hunk 6]
```diff
@@ -83,39 +83,19 @@ you know where to look for certain things.
...
-pages = {295-320},
-year = {2017},
-doi = {10.1137/15M1020575},
+following paper ([preprint](https://arxiv.org/abs/2206.03866)):
+
+```bibtex
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Benoît Legat and Juan Pablo Vielma},
+    title = {JuMP 1.0: Recent improvements to a modeling language for mathematical optimization},
+    journal = {Mathematical Programming C},
```
Comment (odow) [inline docs/src/index.md:92]:
```suggestion
    journal = {Mathematical Programming Computation},
```

File: docs/src/index.md [hunk 7]
```diff
@@ -83,39 +83,19 @@ you know where to look for certain things.
...
+    year = {2023},
+    note = {To appear.},
+    doi = {10.48550/arXiv.2206.03866}
```
Comment (odow) [inline docs/src/index.md:95]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -52,38 +52,48 @@ feature requests, etc.
...
-    pages = {295-320},
-    year = {2017},
-    doi = {10.1137/15M1020575},
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Benoît Legat and Juan Pablo Vielma},
+    title = {JuMP 1.0: Recent improvements to a modeling language for mathematical optimization},
+    journal = {Mathematical Programming C},
+    year = {2023},
+    note = {To appear.},
+    doi = {10.48550/arXiv.2206.03866}
```
Comment (odow) [inline README.md:64]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -52,38 +52,48 @@ feature requests, etc.
...
-    number = {2},
-    pages = {295-320},
-    year = {2017},
-    doi = {10.1137/15M1020575},
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Benoît Legat and Juan Pablo Vielma},
+    title = {JuMP 1.0: Recent improvements to a modeling language for mathematical optimization},
+    journal = {Mathematical Programming C},
+    year = {2023},
+    note = {To appear.},
```
Comment (odow) [inline README.md:63]:
```suggestion
    note = {To appear.}
```

File: CITATION.bib [hunk 10]
```diff
@@ -1,10 +1,8 @@
...
-  pages={295--320},
-  year={2017},
-  publisher={SIAM}
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Benoît Legat and Juan Pablo Vielma},
+    title = {JuMP 1.0: Recent improvements to a modeling language for mathematical optimization},
+    journal = {Mathematical Programming Computation},
+    year = {2023},
+    note = {In press.}
+    doi = {10.48550/arXiv.2206.03866}
```
Comment (odow) [inline CITATION.bib:7]:
```suggestion
```

File: CITATION.bib [hunk 11]
```diff
@@ -1,10 +1,7 @@
...
-  title={JuMP: A modeling language for mathematical optimization},
-  author={Dunning, Iain and Huchette, Joey and Lubin, Miles},
-  journal={SIAM Review},
-  volume={59},
-  number={2},
-  pages={295--320},
-  year={2017},
-  publisher={SIAM}
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Benoît Legat and Juan Pablo Vielma},
```
Comment (blegat) [inline CITATION.bib:2]:
```suggestion
    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Beno{\^i}t Legat and Juan Pablo Vielma},
```

Comment (blegat) [reply CITATION.bib:2]: See https://tex.stackexchange.com/questions/57743/how-to-write-%c3%a4-and-other-umlauts-and-accented-letters-in-bibliography
Some versions of LaTeX and use of inputenc, etc... may make it possible to make it work with "î" directly but my experience is that it's best to stay within the ASCII realm ^^

File: docs/src/index.md [hunk 12]
```diff
@@ -83,39 +83,18 @@ you know where to look for certain things.
...
-volume = {59},
-number = {2},
-pages = {295-320},
-year = {2017},
-doi = {10.1137/15M1020575},
+following paper ([preprint](https://arxiv.org/abs/2206.03866)):
+
+```bibtex
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Benoît Legat and Juan Pablo Vielma},
```
Comment (blegat) [inline docs/src/index.md:90]:
```suggestion
    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Beno{\^i}t Legat and Juan Pablo Vielma},
```

**Review by mlubin — COMMENTED** [review]
Let's just wait to merge until the latest revision shows up on arxiv.

**Comment by odow** [comment]
> Let's just wait to merge until the latest revision shows up on arxiv.

Yes, I see

image

**Comment by odow** [comment]
arXiv link is up: https://arxiv.org/abs/2206.03866

File: README.md [hunk 13]
```diff
@@ -52,38 +52,47 @@ feature requests, etc.
...
-    number = {2},
-    pages = {295-320},
-    year = {2017},
-    doi = {10.1137/15M1020575},
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Beno{\^i}t Legat and Juan Pablo Vielma},
+    title = {JuMP 1.0: Recent improvements to a modeling language for mathematical optimization},
+    journal = {Mathematical Programming C},
+    year = {2023},
+    note = {To appear.}
```
Comment (odow) [inline README.md:63]:
```suggestion
    note = {In press.}
```

File: README.md [hunk 14]
```diff
@@ -52,38 +52,47 @@ feature requests, etc.
...
-    journal = {SIAM Review},
-    volume = {59},
-    number = {2},
-    pages = {295-320},
-    year = {2017},
-    doi = {10.1137/15M1020575},
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Beno{\^i}t Legat and Juan Pablo Vielma},
+    title = {JuMP 1.0: Recent improvements to a modeling language for mathematical optimization},
+    journal = {Mathematical Programming C},
```
Comment (odow) [inline README.md:61]:
```suggestion
    journal = {Mathematical Programming Computation},
```

**Pull request merged**